.github/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Release 0.2.0 (development release)
 
+### Improvements
+
+* A pair of GitHub Actions workflows have been added to generate pull requests
+  for pre-release and post-release version bumps, respectively.
+  [(#12)](https://github.com/XanaduAI/xir/pull/12)
```

Comment:
Is it worth keeping track of changes that don't affect the released package itself here? This change isn't something that concerns any non-developer users nor the downloadable package on PyPI. This includes changes relating to tests and CI. 🤔

Reply:
Honestly, I'm not sure! I figured this is something an XIR contributor would want to know (but perhaps not a user).

I'll remove the comment.

Reply:
I'm leaning towards _not_ having these things in the changelog, but I'm not sure which is best either to be honest.